tests/Unit/Evolution/Systems/CurvedScalarWave/Worldtube/Test_Tags.cpp

```diff
@@ -296,31 +296,49 @@ void test_compute_face_coordinates() {
   }
 }
 
-void test_inertial_particle_position_compute() {
+void test_particle_position_velocity_compute() {
   static constexpr size_t Dim = 3;
   MAKE_GENERATOR(gen);
-  std::uniform_real_distribution<> dist(-10., 10.);
-  const tnsr::I<double, Dim, Frame::Grid> grid_coords_center{
-      {dist(gen), dist(gen), 0.}};
-  const double orbit_radius = get(magnitude(grid_coords_center));
+  const double orbit_radius = 9.;
   const double angular_velocity = 1. / (sqrt(orbit_radius) * orbit_radius);
-  const ::ExcisionSphere<Dim> excision_sphere{2., grid_coords_center, {}};
+  const auto domain_creator =
+      TestHelpers::CurvedScalarWave::Worldtube::worldtube_binary_compact_object(
+          orbit_radius, 0.2, angular_velocity);
   const double initial_time = 0.;
-  auto box = db::create<
-      db::AddSimpleTags<Tags::ExcisionSphere<Dim>, ::Tags::Time>,
-      db::AddComputeTags<Tags::InertialParticlePositionCompute<Dim>>>(
-      excision_sphere, initial_time);
+  auto domain = domain_creator->create_domain();
+  const auto excision_sphere = domain.excision_spheres().at("ExcisionSphereA");
+  std::uniform_real_distribution<> time_dist(0., 1e2);
+  auto box =
+      db::create<db::AddSimpleTags<::Tags::Time, Tags::ExcisionSphere<3>,
+                                   domain::Tags::FunctionsOfTimeInitialize>,
+                 db::AddComputeTags<Tags::ParticlePositionVelocityCompute<3>>>(
+          initial_time, excision_sphere, domain_creator->functions_of_time());
+  const auto get_position_velocity_circular = [orbit_radius, angular_velocity](
+                                                  const double time) {
+    tnsr::I<double, Dim> position{{orbit_radius * cos(angular_velocity * time),
+                                   orbit_radius * sin(angular_velocity * time),
+                                   0.}};
+    tnsr::I<double, Dim> velocity{
+        {-orbit_radius * angular_velocity * sin(angular_velocity * time),
+         orbit_radius * angular_velocity * cos(angular_velocity * time), 0.}};
+    return std::array<tnsr::I<double, Dim>, 2>{std::move(position),
+                                               std::move(velocity)};
+  };
+  // There is an error introduced here which presumable comes from the
+  // quaternion function of time integrating the orbit
+  Approx apprx = Approx::custom().epsilon(1e-8).scale(1.0);
```

Comment:
Yes it's because it's trying to integrate an ODE from 0 to a large time which the quaternion is changing an appreciable amount. To get a more accurate answer, limit the time distribution from 0 to 1.

[Nitpick] Also typo: `presumable` -> `presumably`.